### ISSUE TITLE
Motivation: what do we mean about avoiding breakage for Py2?

### ISSUE THREAD

**Issue by takluyver** [python3statement/pycon-2017#2]
As it stands, I think many people skimming the abstract will say "Releasing Py3-only packages breaks Py2? Well, duh!"

We need to make sure it's clear what we mean, because obviously code that requires Python 3 can't work on Python 2. Instead of talking about 'not breaking' Python 2, we need to say something like "ensuring that users still on Python 2 can easily install an older version, or a parallel 'LTS' version, which will still work for them".

Then there are a few strategies we can talk about:

1. Insist on Py2 users running `pip install foo<5`
2. Use a new package name, so Py3 users run `pip install foo5`
3. The metapackage trick, so `foo` depends on particular versions of `foo_real`.
4. Using the new metadata and machinery we've worked on so none of 1-3 are necessary.

**Comment by Carreau** [comment]
Yes you are right. Thanks for bringing that up and for the english pass.

**Comment by Carreau** [comment]
I've pushed some changes, if you and @michaelpacer want to give it  a read.

**Comment by Carreau** [comment]
Considering adding:

"we <3 users of Python <3"

**Comment by mpacer** [comment]
cute idea for a slide's title in the overview at the end of the proposal. We should have something toward the beginning that talks about our commitment to Python 2 folk and the way in which we want to support them while moving forward. This would be a good title for that slide
